internal/services/containers/container_group_resource.go

```diff
@@ -291,6 +291,41 @@ func resourceContainerGroup() *pluginsdk.Resource {
 							},
 						},
 
+						"cpu_limit": {
+							Type:     pluginsdk.TypeFloat,
+							Optional: true,
+						},
+
+						"memory_limit": {
+							Type:     pluginsdk.TypeFloat,
+							Optional: true,
+						},
```

Comment:
we should be validaing these? also waht sort of limit? we should include the units in the property names

Reply:
The naming of these properties follow the convention as the non limit ones: https://github.com/hashicorp/terraform-provider-azurerm/blob/c268eea425c73e286206ed816cec4f2a8ad8dc08/internal/services/containers/container_group_resource.go#L249-L292

The unit of them are described in the document.